image/dicom_le.ksy

```diff
@@ -0,0 +1,147 @@
+meta:
+  id: dicom
+  file-extension: dcm
+  endian: le
```

Comment:
As I've mentioned, we totally need `/meta/license` here. Also it would be very helpful to add `/meta/title` with official full format name, `/doc-ref` and, if possible, a few more detailed paragraphs of `/doc` — answering general questions about what this format is for, how is it related to other formats, where it is used, where can one obtain samples of the data in this format.